images/base/Dockerfile

```diff
@@ -92,6 +92,11 @@ RUN export ARCH=$(dpkg --print-architecture) \
     && tar -C /opt/cni/bin -xzf /tmp/cni.tgz \
     && rm -rf /tmp/cni.tgz
 
+# Install crictl to /usr/local/bin
+ARG CRICTL_VERSION="v1.13.0"
+RUN export ARCH=$(dpkg --print-architecture) \
+    && curl -fSL "https://github.com/kubernetes-sigs/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz" | tar xzC /usr/local/bin
```

Comment:
I have a feeling we will also need `ctr` cc @Random-Liu

Reply:
i agree that we will need `ctr`, will rework this

/hold